tests/interfaces/records/handlers/records-delete.spec.ts

```diff
@@ -132,14 +132,14 @@ describe('RecordsDeleteHandler.handle()', () => {
     });
 
     it('should only write the data once even if written by multiple messages', async () => {
```

Comment:
Not really relevant to the PR, but do you know why we have this test inside `handlers/records-delete.spec.ts`? Doesn't look like we use `RecordsDelete` or do deleting of any kind in this test.